test/python/transpiler/test_depth_pass.py

```diff
@@ -71,6 +71,27 @@ def test_depth_one(self):
 
         self.assertEqual(pass_.property_set["depth"], 1)
 
+    def test_depth_control_flow(self):
+        """A DAG with control flow still gives an estimate."""
+        qc = QuantumCircuit(5, 1)
+        qc.h(0)
+        qc.measure(0, 0)
+        with qc.if_test((qc.clbits[0], True)) as else_:
+            qc.x(1)
+            qc.cx(2, 3)
+        with else_:
+            qc.x(1)
+            with qc.for_loop(range(3)):
+                qc.z(2)
+                with qc.for_loop((4, 0, 1)):
+                    qc.z(2)
+        with qc.while_loop((qc.clbits[0], True)):
+            qc.h(0)
+            qc.measure(0, 0)
+        pass_ = Depth()
+        pass_(qc)
+        self.assertEqual(pass_.property_set["depth"], 16)
```

Comment:
A comment here about how to arrive at the value 16 would be helpful in parsing these types of tests.

Reply:
Done in e231e7e96.